DEVELOP.md

```diff
@@ -53,38 +105,29 @@ Install the latest version of QGIS 3 using [these](https://gis4wrf.github.io/ins
 
 QGIS 3 on macOS uses Python installed in `/usr/local`. Python packages can be installed using pip by invoking the Python 3 interpreter on the system: `python3 -m pip install 'package-name'`.
 
-## ZIP packaging
-
-To create the plugin archive simply run `python build.py`.
-
-## Package structure
-
-- `gis4wrf.core` contains all non-QGIS-dependent code
-- `gis4wrf.qgis` is the QGIS plugin and uses `gis4wrf.core`
-
-## External packages
-
-gis4wrf uses several external Python packages to read, write, and visualize data.
-Some are available with every QGIS installation, like GDAL, while others are installed
-separately, like wrf-python and netCDF4. See `gis4wrf/bootstrap.py` for more details
-on handling external packages.
-
-### A note on GDAL
 
-QGIS comes with GDAL's Python bindings which are very close to its C interface and hard to use.
-Ideally we'd like to use the [rasterio](https://mapbox.github.io/rasterio) library
-which is a more pythonic wrapper for GDAL, however this library [cannot be used](https://rasterio.readthedocs.io/en/latest/topics/switch.html#mutual-incompatibilities) together with
-GDAL's Python bindings in the same program, here QGIS. 
+# Release
 
-## Useful websites
+## ZIP packaging
 
-- [Python GDAL/OGR Cookbook](https://pcjericks.github.io/py-gdalogr-cookbook/)
+To create the plugin archive simply run `python build.py`.
 
-## License header
+## Prepare a new release
 
-The following header shall be appended at the top of every source file.
+Before creating a new release, make sure to complete the following checklist:
 
 ```
-# GIS4WRF (https://doi.org/10.5281/zenodo.1288569)
-# Copyright (c) 2018 D. Meyer and M. Riechert. Licensed under MIT.
-```
+- [ ] Create a new issue named `Prepare for GIS4WRF <VERSION_NUMBER> release`, and copy and paste this check-list and tick what is appropriate.
+- [ ] Check that the [documentation](https://gis4wrf.github.io/) repository is up to date.
+- [ ] Check that the [readme](README.md) file is up-to-date.
+- [ ] Check that the [metadata.txt](gis4wrf/metadata.txt) file is up-to-date.
+- [ ] Create a milestone for the version to release.
```

Comment:
This should be done way before releasing. At this point it's more like "if not done already"

Reply:
Changed to  "Create a milestone for the version to release if not done already" and moved up